src/main/java/com/redhat/vertx/pipeline/Section.java

```diff
@@ -47,7 +47,7 @@ private Step buildStep(JsonObject def) {
             throw new RuntimeException("Unknown keys in configuration: " + defKeys.toString());
         }
 
-        // We should only have one entry, use that for the sort name to class mapping
+        // We should only have one entry, use that for the short name to class mapping
```

Comment:
We may want to pull this out of `Section` as well, though we could do that at a later point in time.